### PR TITLE
Update min version Serilog for usage in Asp.net 5 project

### PR DESCRIPTION
**Pull request by Sensarg22**
issue #3

### CODE REVIEW

File: src/Serilog.Sinks.AzureTableStorage/packages.config [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 <packages>
-  <package id="Microsoft.Data.Edm" version="5.6.3" targetFramework="net45" />
-  <package id="Microsoft.Data.OData" version="5.6.3" targetFramework="net45" />
-  <package id="Microsoft.Data.Services.Client" version="5.6.3" targetFramework="net45" />
-  <package id="Microsoft.WindowsAzure.ConfigurationManager" version="2.0.3" targetFramework="net45" />
-  <package id="Newtonsoft.Json" version="6.0.6" targetFramework="net45" />
-  <package id="Serilog" version="1.4.204" targetFramework="net45" />
-  <package id="System.Spatial" version="5.6.3" targetFramework="net45" />
-  <package id="WindowsAzure.Storage" version="4.3.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
```
Comment (robdmoore) [inline src/Serilog.Sinks.AzureTableStorage/packages.config]: Why is this package in there?

**Comment by robdmoore** [comment]
#3 mentions .NET core.

If you are trying to use this with .NET core then I don't think this will work since it's still referencing dlls from and building against .NET4.5.

I'm not an expert in .NET core though so happy to be proven wrong.

**Comment by Sensarg22** [comment]
You are right with the .net core will not work, but will work with dnx451 within asp.net 5

File: src/Serilog.Sinks.AzureTableStorage/packages.config (hunk 1 above)
Comment (robdmoore) [inline src/Serilog.Sinks.AzureTableStorage/packages.config]: I know why now - that later versions of storage SDK depend on it.

**Comment by robdmoore** [comment]
I'm happy with this if you are @nblumhardt.

I note the serilog version it depends on is a beta though - is that an issue?

How do we make sure there is a major version bump in this library or does the build server automatically match it to the serilog version?

**Comment by nblumhardt** [comment]
We'll need to open a `dev` branch in this repo; I'll pull the PR into it when I have a chance later today :+1:

**Comment by nblumhardt** [comment]
This is merged to dev, I'm working on getting a 2.0.0-beta* build out of it. The PR won't show up as merged until it hits master; I'm changing the repo's default branch now to use dev for future PRs.

**Comment by nblumhardt** [comment]
Done! 2.0.0-beta-20 is now on NuGet :-)

**Comment by robdmoore** [comment]
Nice!

**Pull request merged**